spec/features/components_guide_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.feature "Visit Component Guide" do
+  include ComponentGuideSteps
+
+  describe "Component Guide" do
+    scenario "User views the component guide page" do
+      when_i_visit_the_component_guide_index_page
+      i_see_a_page_title
+      i_see_subheadings
```

Comment:
These two assertion seem to actually test parts of govuk_publishing_components gem by checking for HTML and text which the gem provides. 

I'd probably just do:
```
  when_i_visit_the_component_guide_index_page
  then_i_receive_a_valid_response
end

# ...

def then_i_receive_a_valid_response
  expect(page.status_code).to eq(200)
end
```

Reply:
Done